src/guides/v2.3/comp-mgr/upgrade-best-practices.md

```diff
@@ -0,0 +1,94 @@
+---
+group: software-update-guide
+title: Upgrade best practices
+functional_areas:
+  - Upgrade
+---
+
+This topic lists the actions you should take to manage the complexity of upgrading Magento Open Source and {{site.data.var.ee}}. Your team should be thinking about upgrades from the moment your project development starts and continue through each release. By following these best practices, the upgrade process will be much easier, faster, and cheaper.
+
+{:.bs-callout-tip}
+These recommendations are based on best practices supported by evidence for its impact and effectiveness from partners, merchants, Adobe experts, and the community.
+
+## What impacts an upgrade?
+
+It is important to understand the variables that determine the complexity of an upgrade. You must consider these variables at the beginning of each project--not just when is time to upgrade. The development of a project is key to ensure future upgrades will be smooth and that you will be able to control the required effort to complete them.
+
+The level of effort to upgrade your {{site.data.var.ee}} instance depends upon these factors:
+
+*  **How you have built your site?** The amount of custom work and the number of installed 3rd-party modules strongly affects the complexity of an upgrade. The quality of the custom work and modules also can also determine whether an upgrade will go smoothly.
+
+*  **Are you skipping multiple releases?** Skipping releases will makes the next upgrade more complex, upgrading from consequent versions will make the process easier and cheaper.
+
+*  **What type of upgrade are you performing?** An upgrade to a minor release (from 2.3.x to 2.4.0, for example) is more extensive than an upgrade between patch releases (such as from 2.4.2 to 2.4.3). Security upgrades are the easiest type to implement.
+
+## What should an upgrade cost?
+
+This is the question you should be asking yourself and your team when starting a project. If your project is running already, then how much are upgrades costing you? Is this the regular upgrade cost expected for {{site.data.var.ee}}?
+
+The upgrade cost may vary significantly from project to project, based on the variables that impact upgrades, as described previously. Based on our experience, you should plan for upgrades to take the following amounts of time:
+
+Upgrade type | Example versions | Hours
+--- | ---
+Security and patchupgrades | 2.4.3-p1 to 2.4.3-p2 | 16 - 24
+Minor release upgrades | 2.3.x to 2.4.x | 20 - 50
+Major release upgrades | 1.x to 2.x | 500 - 600
```

Comment:
This should be 2.3.x to 2.4.x for example, because those are major releases.

If you want to also mention upgrading from Magento 1 to Magento 2, it should be on a 4th line, and that's actually considered a migration instead of an upgrade because it's a very different system.